api/src/main/java/ca/bc/gov/educ/api/distribution/model/dto/v2/School.java

```diff
@@ -0,0 +1,67 @@
+package ca.bc.gov.educ.api.distribution.model.dto.v2;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+public class School {
+
+	private String minCode;
+	private String schoolId;
+	private String schoolName;
+	private String districtName;
+	private String transcriptEligibility;
+	private String certificateEligibility;
+	private String address1;
+	private String address2;
+	private String city;
+	private String provCode;
+	private String countryCode;
+	private String postal;
+	private String openFlag;
+	private String schoolCategoryCode;
+	private String schoolCategoryCodeInstitute;
+
+	@JsonIgnore
+	private String schlNo;
+	@JsonIgnore
+	private String distNo;
+    
+	public String getSchoolName() {
+		return  schoolName != null ? schoolName.trim(): null;
```

Comment:
Is the trim necessary? Could just use lombok Getter

Reply:
Good catch.  I think it is copied from TraxSchool as it's open vms database fills space if the actual value is less than the field size defined in the table.  I will update it soon.